docs/ldoc.css

```diff
@@ -0,0 +1,298 @@
+body { 
+    color: #47555c;
+    font-size: 16px;
+    font-family: "Open Sans", sans-serif;
```

Comment:
why not just 'Sans' to use the system font?

and the same for monospace above

Reply:
Makes sense to me.
Should be forwarded to https://github.com/stevedonovan/LDoc/pull/287 then.

Reply:
copied the comment there

Reply:
Thanks.

Does 9480706 address your comment?